wiki/Help_centre/id.md

```diff
@@ -10,70 +10,70 @@ tags:
   - masalah
   - permasalahan
   - kehilangan
-outdated_translation: true
-outdated_since: 737ea7fc657bed12687ef10e79bb9ae8f444c465
 ---
 
 # Pusat bantuan
 
-Apakah kamu mengalami suatu masalah? Kami siap untuk membantumu! Pada *sidebar* yang tertera pada sisi kiri layar, kamu dapat melihat daftar masalah-masalah yang umum dihadapi beserta solusinya. Apabila masalahmu tidak tertera pada artikel ini atau semakin menjadi-jadi seiring waktunya, harap hubungi kami melalui email pada alamat [support@ppy.sh](mailto:support@ppy.sh). Di samping itu, kamu juga dapat meminta bantuan kepada para pengguna lainnya melalui [sub-forum Help](https://osu.ppy.sh/community/forums/5).
+Apakah kamu menemui suatu masalah? Kami siap untuk membantumu! Pada artikel ini, kamu akan dapat mempelajari berbagai masalah yang umum dihadapi beserta solusinya. Apabila masalahmu tidak tertera di bawah ini atau semakin menjadi-jadi seiring waktunya, harap hubungi kami melalui email pada alamat [support@ppy.sh](mailto:support@ppy.sh). Di samping itu, kamu juga dapat meminta bantuan kepada para pengguna lainnya melalui [sub-forum Help](https://osu.ppy.sh/community/forums/5).
 
 ## Jenis permasalahan {#sections}
 
 Harap pilih jenis permasalahan yang paling menggambarkan masalah yang kamu hadapi dari daftar di bawah ini agar kami dapat membantumu lebih lanjut.
 
-| Jenis permasalahan | Topik-topik yang terlingkup |
+| Jenis permasalahan | Topik yang terlingkup |
 | :-- | :-- |
-| [Akun](/wiki/Help_centre/Account) | osu!supporter, pembatasan akun (*restriction*), proses *login*, penggantian nama pengguna, data profil |
+| [Akun](/wiki/Help_centre/Account) | osu!supporter, proses *login* dan pemulihan akun, penggantian nama pengguna, data profil |
+| [Restriction](/wiki/Help_centre/Account_restrictions) | Segala hal seputar pembatasan akun (*account restriction*): garis besar, proses banding, alasan umum, masa hukuman |
```

Comment:
```suggestion
| [Restriksi](/wiki/Help_centre/Account_restrictions) | Segala hal seputar pembatasan akun (*account restriction*): garis besar, proses banding, alasan umum, masa hukuman |
```

Reply:
`restriksi` aka `restriks` is used common in indonesia

Reply:
ah, this is intentional and has been discussed in http://github.com/ppy/osu-wiki/pull/5703#discussion_r654817584 among others fwiw...

(not to mention that http://osu.ppy.sh/wiki/id/Help_centre/Account_restrictions also has "Restriction" instead of "Pembatasan" as its title)